server/services/team/service.ts

```diff
@@ -159,4 +159,8 @@ export class TeamService extends ServiceClass implements ITeamService {
 
 		return this.TeamMembersModel.findByTeamId(teamId).toArray();
 	}
+
+	async update(teamId: string, updateData: ITeamUpdate): Promise<void> {
+		this.TeamModel.updateOne({ _id: teamId }, updateData);
```

Comment:
This will allow users to update the `roomId` property (which identifies the team's main room) but the changes aren't being propagated to the room instance 👀

Can you add a conditional that, if the roomId is modified, then the room should also be updated accordingly?

Reply:
at this point, I don't think we should allow changing the `roomId` from a team.. looking at the endpoint payload, it looks like it doesn't accept a `roomId` though, which is good.

My change request would actually be to change this model call to a specific method, something like `this.TeamModel.updateNameAndTypeById(teamId, updateData)`, and there use an actual `$set` operation, because this one is actually overwriting everything on the document.

Reply:
Agreed